feature-libs/cart/main/root/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -149,36 +132,44 @@ export class AddToCartComponent implements OnInit, OnDestroy {
     if (!this.productCode || quantity <= 0) {
       return;
     }
+
+    let numberOfEntriesBeforeAdd = 0;
```

Comment:
we don't need this variable, right? just pass `entries.length` to `createCartUiEventAddToCart`.

Reply:
There was a class property before and I made it a local var instead, but perhaps we don't event need that variable.  I'll change iit